files/acl.py

```diff
@@ -226,79 +238,72 @@ def main():
     state = module.params.get('state')
     follow = module.params.get('follow')
     default = module.params.get('default')
-
```

Comment:
Done in f8027a7.

On one hand, I am very happy we now support recursive ACLs, on the other hand I am very sad we are dropping support for BSD because of this.

Note however that there are some artifacts left: the `-h`option (now on [line 179](https://github.com/ansible/ansible-modules-core/pull/150/files#diff-9c3c3b66bc72c68c9eafd9cefa084c71R179)) is BSD-only while the equivalent must be set with `-L`/`-P` for Linux. See the [Linux man](http://linux.die.net/man/1/setfacl) and the [FreeBSD man](http://www.freebsd.org/cgi/man.cgi?format=html&query=setfacl(1)) for reference.

I'd be more than happy to give BSD its `acl` module back and to fix that issue described above once this PR makes it upstream. I'm afraid it will be forever delayed if making them part of this PR.

What do you think of my suggestions in the "Suggestions for Linux / BSD support" section of this PR description?